source/js/form.js

```diff
@@ -0,0 +1,43 @@
+(function() {
```

Comment:
Code does not pass lint rules; `npm run lint`
(Rules in `/ExpressBoilerplate/.eslintrc.json` must be updated)

<img width="924" alt="ESLint errors" src="https://user-images.githubusercontent.com/1402711/36875455-2805b30e-1e04-11e8-82bf-39cf1f9623c7.png">